src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -281,20 +281,38 @@
     (future-cancel command-loader))
   context)
 
-(defn initialize-schema
-  "Ensure the database is migrated to the latest version, and warn if
-  it's deprecated, log and exit if it's unsupported.
-
-  This function will return true iff any migrations were run."
-  [db-conn-pool config]
+(defn db-unsupported-msg
+  "Returns a message describing which databases are supported."
+  []
+  (str (trs "PostgreSQL DB versions older than {0}.{1} are no longer supported."
```

Comment:
We shouldn't glue trs-ed strings together, instead, there should be one huge string inside a single TRS.